java/spacewalk-java.changes.rrichardson.reposync_package_deletion

```diff
@@ -0,0 +1 @@
+- Add no-strict checkbox to manage channel sync page
```

Comment:
Think like the admin which read this changelog. What would you like to know about this change?

Reply:
Changed to `Implement new default behaviour for channel synchronisation, now automatically dropping packages which are deleted from the upstream repository (strict mode)` ?